packages/web-components/src/components/ic-button/ic-button.tsx

```diff
@@ -174,10 +174,7 @@ export class Button {
 
   private hasIconSlot(): boolean {
     const iconEl = this.el.querySelector(`[slot="icon"]`);
-    if (iconEl != null) {
-      return true;
-    }
-    return false;
+    return iconEl !== null ? true : false;
```

Comment:
Does this need true or false after since if it is not equal, it will return true and if not false anyway?